src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -374,18 +372,42 @@ public WebElement sectionLevelPanel(int instrNum, int sectionLevelIndex) {
         return browser.driver.findElement(By.id(permissionDivId));
     }
     
+    private WebElement sectionLevelPanelBody(int instrNum, int sectionLevelIndex) {
+        WebElement sectionPanel = this.sectionLevelPanel(instrNum, sectionLevelIndex);
+        String cssSelector = "div[class='panel-body']";
+        return sectionPanel.findElement(By.cssSelector(cssSelector));
+    }
+    
     private WebElement sectionSelectionCheckBox(int instrNum, int sectionLevelIndex, int sectionNum) {
         WebElement sectionPanel = this.sectionLevelPanel(instrNum, sectionLevelIndex);
         String cssSelector = "input[name='" + Const.ParamsNames.INSTRUCTOR_SECTION_GROUP
                              + sectionLevelIndex + Const.ParamsNames.INSTRUCTOR_SECTION + sectionNum + "']";
         return sectionPanel.findElement(By.cssSelector(cssSelector));
     }
     
-    private WebElement sectionLevelPanelCheckBox(int instrNum, int sectionLevelIndex, String checkBoxName) {
-        WebElement sectionPanel = this.sectionLevelPanel(instrNum, sectionLevelIndex);
-        String cssSelector = "input[type='checkbox'][name='" + checkBoxName
-                             + Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + sectionLevelIndex + "']";
-        return sectionPanel.findElement(By.cssSelector(cssSelector));
+    private WebElement sectionLevelPanelCheckBox(int instrNum, int sectionLevelIndex, int checkBoxIndex) {
+        WebElement sectionPanelBody = this.sectionLevelPanelBody(instrNum, sectionLevelIndex);
+        String cssSelector = "input[type='checkbox']";
+        return sectionPanelBody.findElements(By.cssSelector(cssSelector)).get(checkBoxIndex);
```

Comment:
Changed the earlier methods with a `WebElement` return type to use `get...` as well